feature/tunnel/otg_tests/tunnel_interface_based_resize_test/tunnel_interface_based_resize_test.go

```diff
@@ -0,0 +1,560 @@
+package tunnel_interface_based_resize_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/cfgplugins"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	otgconfighelpers "github.com/openconfig/featureprofiles/internal/otg_helpers/otg_config_helpers"
+	otgvalidationhelpers "github.com/openconfig/featureprofiles/internal/otg_helpers/otg_validation_helpers"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/netutil"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// TestMain calls main function.
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+const (
+	ieee8023adLag   = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+	trafficDuration = 20 * time.Second
+)
+
+var (
+	top             = gosnappi.NewConfig()
+	aggID           string
+	egressLag1Ports = []string{"port2", "port3"}
+	egressLag2Ports = []string{"port4", "port5"}
+	staticRoute     = "10.99.20.0"
+
+	ingressIntf = attrs.Attributes{
+		Desc:    "ingress_interface",
+		IPv4:    "194.0.2.1",
+		IPv4Len: 30,
+		IPv6:    "2001:10:1:6::1",
+		IPv6Len: 126,
+	}
+	egressLag1Intf = attrs.Attributes{
+		Desc:    "Egress_Lag1_Intf",
+		IPv4:    "169.254.0.1",
+		IPv6:    "2000:0:0:2::1",
+		MTU:     9202,
+		IPv4Len: 30,
+		IPv6Len: 126,
+	}
+	egressLag2Intf = attrs.Attributes{
+		Desc:    "Egress_Lag2_Intf",
+		IPv4:    "169.254.0.5",
+		IPv6:    "2000:0:0:2::5",
+		MTU:     9202,
+		IPv4Len: 30,
+		IPv6Len: 126,
+	}
```

Comment:
https://google.github.io/styleguide/go/decisions#initialisms
>Words in names that are initialisms or acronyms (e.g., URL and NATO) should have the same case.

`Lag` -> `LAG`


```suggestion
	egressLAG1Ports = []string{"port2", "port3"}
	egressLAG2Ports = []string{"port4", "port5"}
	staticRoute     = "10.99.20.0"

	ingressIntf = attrs.Attributes{
		Desc:    "ingress_interface",
		IPv4:    "194.0.2.1",
		IPv4Len: 30,
		IPv6:    "2001:10:1:6::1",
		IPv6Len: 126,
	}
	egressLAG1Intf = attrs.Attributes{
		Desc:    "Egress_Lag1_Intf",
		IPv4:    "169.254.0.1",
		IPv6:    "2000:0:0:2::1",
		MTU:     9202,
		IPv4Len: 30,
		IPv6Len: 126,
	}
	egressLAG2Intf = attrs.Attributes{
		Desc:    "Egress_Lag2_Intf",
		IPv4:    "169.254.0.5",
		IPv6:    "2000:0:0:2::5",
		MTU:     9202,
		IPv4Len: 30,
		IPv6Len: 126,
	}
```